app/views/events/new.html.erb

```diff
@@ -0,0 +1,20 @@
+<div class="row">
+  <div class="col-md-6 form-new-user col-md-offset-3">
+    <% if logged_in? %>
+      <%= form_for(@event) do |f| %>
+        <%= f.label :title %><br>
+        <%= f.text_area :title, class: "form-control" %><br>
+        <%= f.label :description %><br>
+        <%= f.text_area :description, class: "form-control" %><br>
+        <%= f.label :location %><br>
+        <%= f.text_area :location, class: "form-control" %><br>
+        <%= f.label :date %><br>
+        <%= f.date_field :date, class: "form-control" %><br>
+        <%= f.submit 'Create', class: 'btn btn-primary' %>
+      <% end %>
+    <% else %>
+      <h3> You are not logged, try to <%= link_to "Log in here!", login_path =%> <h3/><br>
```

Comment:
this is not the best implementation, its best to actually redirect to the login screen.
if we had 10 screens, would you do this for all the pages? its too much work.

Reply:
Fixed by add `before_action :logged_in_user, only: [:new]`
and define method `#logged_in_user` in `Events` controller